factorio/factorio/egg-pterodactyl-factorio-a-r-m64.json

```
@@ -4,19 +4,20 @@
         "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2024-09-08T19:58:10+02:00",
+    "exported_at": "2025-09-21T19:35:14+00:00",
     "name": "Factorio ARM64",
     "author": "parker@parkervcp.com",
     "description": "The vanilla Factorio server.\r\n\r\nhttps:\/\/www.factorio.com\/",
-    "features": null,
+    "tags": [],
+    "features": [],
```

Ptero expects this to be null if no features are present
```suggestion
    "features": null,
```

---

@rmartinoscar It imported fine with no issues on my end